recipe/meta.yaml

```diff
@@ -34,8 +34,7 @@ outputs:
         - dill
         - h5py
         - ipython
-        - jupyter
```

Comment:
The [`jupyter`](https://github.com/conda-forge/jupyter-feedstock/blob/master/recipe/meta.yaml) package is actually a metapackage which install notebook, qtconsole, etc... and therefore defies the purpose of this PR!